.github/workflows/downstream-ci.yml

```diff
@@ -330,7 +330,7 @@ jobs:
       uses: actions/checkout@v4
       with:
         repository: ecmwf/downstream-ci
-        ref: main
+        ref: feature/add_pymultio
```

Comment:
(to be complete, same comment here)

Reply:
Yes, this is automated, and will go back to 'main' when we merge